flexget/utils/imdb.py

```diff
@@ -166,7 +168,15 @@ def search(self, name):
             movie = {}
             additional = re.findall(r'\((.*?)\)', row.text)
             if len(additional) > 0:
-                movie['year'] = additional[-1]
+                try:
+                    int(additional[-1])
```

Comment:
I'm not really a fan of this; it's obscure and silly. Why not just check for type?